### PR TITLE
Add page Setting-up-Rhetos-for-HTTPS

### PR DESCRIPTION
**Pull request by glisak**
Hi,

I added a new file Setting-up-Rhetos-for-HTTPS.md

### CODE REVIEW

File: Setting-up-Rhetos-for-HTTPS.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Setting up Rhetos for HTTPS
+
+In order for Rhetos to work on HTTPS you need to add a new endpoint in web.config file.
+
+Insert following code:
+```C
```
Comment (bantolov) [inline Setting-up-Rhetos-for-HTTPS.md]: All snippets here should use XML formatting instead of C.

File: Setting-up-Rhetos-for-HTTPS.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+<services>
+      <service name="Rhetos.RhetosService">
+        <clear />
+        <endpoint address="" binding="basicHttpBinding" bindingConfiguration="rhetosBasicHttpBinding" name="basic" contract="Rhetos.IServerApplication" listenUriMode="Explicit">
+          <identity>
+            <dns value="localhost" />
+            <certificateReference storeName="My" storeLocation="LocalMachine" x509FindType="FindBySubjectDistinguishedName" />
+          </identity>
+        </endpoint>
+    '''<endpoint address="mex" binding="mexHttpsBinding" contract="Rhetos.IServerApplication" />'''
```
Comment (bantolov) [inline Setting-up-Rhetos-for-HTTPS.md]: Please remove all `'''` from the code snippet. Is seems to be an obsolete formatting that somehow got left in the old version of the text.

File: Setting-up-Rhetos-for-HTTPS.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
+# Setting up Rhetos for HTTPS
+
+In order for Rhetos to work on HTTPS you need to add a new endpoint in web.config file.
+
+Insert following code:
```
Comment (bantolov) [inline Setting-up-Rhetos-for-HTTPS.md:5]: This sentence was probably originally written for an old version of Rhetos.
[Web.config](https://github.com/Rhetos/Rhetos/blob/master/Source/Rhetos/Web.config) already contains the `security` elements by default, it should be _updated_, not added.

**Review by bantolov — CHANGES_REQUESTED** [review]

File: Setting-up-Rhetos-for-HTTPS.md (hunk 1 above)
Comment (glisak) [reply Setting-up-Rhetos-for-HTTPS.md]: Changed to XML

File: Setting-up-Rhetos-for-HTTPS.md (hunk 2 above)
Comment (glisak) [reply Setting-up-Rhetos-for-HTTPS.md]: Removed in new version

File: Setting-up-Rhetos-for-HTTPS.md (hunk 3 above)
Comment (glisak) [reply Setting-up-Rhetos-for-HTTPS.md:5]: I'm not sure I understand this. I changed it to "In order for Rhetos to work on HTTPS you need to update security elements in web.config file." Hope this is ok.

**Pull request merged**